store.html

```diff
@@ -0,0 +1,153 @@
+          /></label>
+        </ul>
+      </nav>
+    </header>
+    <main class="storemain">
+      <section class="formsection">
+        <h1>Order your Karma wifi device today!</h1>
+
+        <form action="#">
+          <div class="fieldset">
+            <div class="flex">
+              <label for="firstname">First name *</label>
+              <input type="text" id="firstname" required />
+            </div>
+            <div class="flex">
+              <label for="lastname">Last name *</label>
+              <input type="text" id="lastname" required />
+            </div>
+          </div>
+
+          <div class="address">
+            <div class="flex">
+              <label for="addressone">Address *</label>
+              <input type="text" id="addressone" required />
+            </div>
+          </div>
+
+          <div class="address">
+            <div class="flex">
+              <label for="addresstwo">Address 2</label>
+              <input type="text" id="addresstwo" />
+            </div>
+          </div>
+
+          <div class="fieldset">
+            <div class="flex">
+              <label for="city">City *</label>
+              <select name="city" id="city" required>
+                <option value="" disabled selected>Select your city...</option>
+                <option value="tehran">Tehran</option>
+                <option value="london">London</option>
+                <option value="cardiff">Cardiff</option>
+              </select>
+            </div>
+
+            <div class="flex">
+              <label for="postcode">Postcode *</label>
+              <input type="text" id="postcode" required />
+            </div>
+          </div>
+
+          <div class="radiobox">
+            <legend>Select a color</legend>
+            <ul>
+              <li>
+                <input
+                  type="radio"
+                  id="karmaorange"
+                  name="color"
+                  value="karmaorange"
+                  checked
+                />
+                <label for="karmaorange">Karma Orange</label>
+              </li>
+              <li>
+                <input
+                  type="radio"
+                  id="spacegrey"
+                  name="color"
+                  value="spacegrey"
+                />
+                <label for="karmaorange">Space Grey</label>
+              </li>
+            </ul>
+          </div>
+
+          <div class="checkbox">
+            <input type="checkbox" id="termsandconditions" required />
+            <label for="termsandconditions"
+              >By placing your order you agree to
+              <a href="">Karma's Terms and Conditions</a> *</label
+            >
+          </div>
+
+          <div>
+            <button>Place my order</button>
+          </div>
+        </form>
+      </section>
+
+      <section class="imgsection"></section>
+    </main>
+
+    <footer>
+      <h3>Join us on</h3>
```

Comment:
This should probably be a `h2`. It would be an h3 if it was nested under an existing `h2` section